pkg/sql/colflow/colrpc/outbox.go

```diff
@@ -395,7 +396,7 @@ func (o *Outbox) runWithStream(
 			}
 			switch {
 			case msg.Handshake != nil:
-				log.VEventf(ctx, 2, "Outbox received handshake: %v", msg.Handshake)
+				log.VEventf(ctx, 2, "Outbox received handshake: %v", redact.Safe(msg.Handshake))
```

Comment:
Make handshake handle the Safe formatter interface instead. It's not generally safe.